draft-thomson-webpush-protocol.xml

```diff
@@ -328,7 +338,40 @@ Cache-Control: max-age:864000, private
 ]]></artwork>
       </figure>
     </section>
+    <section anchor="receipt_subscription" title="Subscribing to Receipts">
```

Comment:
Note for later: lets consider restructuring the document now.  I like a relatively flat structure, but I think that we could do a little better.